apiary.apib

```diff
@@ -4,9 +4,42 @@ FORMAT: 1A
 
 The API for interacting with IPFS nodes.
 
-# Group add
+# Group add [GET /add{?arg}{&recursive,quiet,silent,progress,trickle,only%2Dhash,wrap%2Dwith%2Ddirectory,hidden,chunker,pin}]
+Add an object to ipfs.
 
-TODO
+Adds contents of <path> to ipfs. Use -r to add directories.
+Note that directories are added recursively, to form the ipfs
+MerkleDAG. A smarter partial add with a staging area (like git)
+remains to be implemented.
+
++ Parameters
+    + arg (string, required) - The path to a file to be added to IPFS
```

Comment:
@ABISprotocol Not sure why you're bringing that up here. My email address is on my profile, feel free to contact me there.